sqlQueries/queryPollutantByState.sql

```diff
@@ -15,11 +19,15 @@ ORDER BY date_local
 --Returns all samples for a state
```

Comment:
SELECT psample.date_local, psample.id, ptype.name, AVG(psample.mean)
FROM pollutant_sample AS psample
INNER JOIN taken_at AS ta
	ON psample.sample_id = ta.sample_id
INNER JOIN in_state
	ON ta.site_num = in_state.site_num
INNER JOIN is_type AS it
    ON psample.id = it.sample_id
INNER JOIN pollutant_type AS ptype
    ON ptype.id = it.type_id
WHERE in_state.state_code = user_provided_state_code
GROUP BY date_local
ORDER BY ptype.name, date_local`

Reply:
Added this to queryPollutantByState.sql:
--Returns the average of the means of the pollutants across all sites in a state, for each date.
--The data is ordered by date_local and pollutant name which is useful for creating a linegraph
--Inputs (user_provided_state_code)
SELECT psample.date_local, psample.id, ptype.name, AVG(psample.mean)
FROM pollutant_sample AS psample
	INNER JOIN taken_at AS ta
		ON psample.sample_id = ta.sample_id
	INNER JOIN in_state
		ON ta.site_num = in_state.site_num
	INNER JOIN is_type AS it
		ON psample.id = it.sample_id
	INNER JOIN pollutant_type AS ptype
		ON ptype.id = it.type_id
WHERE in_state.state_code = user_provided_state_code
GROUP BY date_local, ptype.name
ORDER BY ptype.name, date_local;

Reply:
I restored queryPollutantByState to what it was before the averages were used, and made averaged versions of those queries in queryPollutantByState_Average.